LA4Ice.c

```diff
@@ -283,7 +283,7 @@ int main(int argc, char *argv[])
   }
 
   //  Read the file and display selected records
-  
+  if (tspace > 0)
```

Comment:
I still don't get it. If `tspace == 0` implies an empty file, then what are you skipping? `novl` is the number of overlap records. What other kind of record could there be? Why not just compare `novl==0`? And are you sure that you get a speed-up? How? Exactly what was happening that doesn't need to happen? Just some extra boilerplate printing?

*tspace* seems like *total space* or *trace space* maybe.